provider/cmd/pulumi-gen-aws-native/main.go

```diff
@@ -34,7 +34,7 @@ import (
 
 func main() {
 	flag.Usage = func() {
-		const usageFormat = "Usage: %s <operation> <schema-folder> <version> (<schema urls>)"
+		const usageFormat = "Usage: %s <operation> <schema-folder> <version> (<schema urls>) <docs-url>"
```

Comment:
A lot of positional arguments, could we use keyword based `-docs-url` flag or some such?

Reply:
ok I refactored this to use flags instead of arguments